include/zcbor_common.h

```diff
@@ -46,12 +46,14 @@ do { \
 #define MIN(a, b) (((a) < (b)) ? (a) : (b))
 #endif
 
+#ifndef ARRAY_SIZE
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
+#endif
 
-struct zcbor_state_backups_s;
 
-typedef struct zcbor_state_backups_s zcbor_state_backups_t;
+struct zcbor_state_constant;
 
-typedef struct{
+typedef struct {
```

Comment:
While we are at it, should we drop the struct typedef here? There are only two left in the repo, `zcbor_state_t` and `zcbor_string_type_t`